website/source/guides/acl.html.markdown

```diff
@@ -0,0 +1,310 @@
+---
+layout: "guides"
+page_title: "ACLs"
+sidebar_current: "guides-acl"
+description: |-
+  Don't panic! This is a critical first step. Depending on your deployment
+  configuration, it may take only a single server failure for cluster
+  unavailability. Recovery requires an operator to intervene, but recovery is
+  straightforward.
+---
+
+# ACL System
+
+Nomad provides an optional Access Control List (ACL) system which can be used to control access to data and APIs. The ACL is [Capability-based](https://en.wikipedia.org/wiki/Capability-based_security), relying on tokens which are associated with policies to determine which fine grained rules can be applied. It is very similar to [AWS IAM](https://aws.amazon.com/iam/) in many ways.
+
+# ACL System Overview
+
+The ACL system is designed to be easy to use, fast to enforce, and flexible to new policies, all while providing administrative insight. At the highest level, there are three major components to the ACL system:
```

Comment:
If you can find better wording to "flexible to new policies" that would be good. Reads a bit funny